index.js

```
@@ -0,0 +1,171 @@
+                console.error("JSON parsing error:", err.message);
+                sendJsonResponse(res, 400, { error: 'Invalid JSON body: ' + err.message });
+                return;
+            }
+            try {
+                const { repositoryUrl, workingDirectory = '' } = data;
+                if (!repositoryUrl) {
+                    sendJsonResponse(res, 400, { error: 'Missing required field: repositoryUrl' });
+                    return;
+                }
+                const result = initializeEnvironment(repositoryUrl, workingDirectory);
+                if (result && result.error) { // Check for error flag from initializeEnvironment
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)